cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -459,7 +552,10 @@ func (g *genConversion) doStruct(inType, outType *types.Type, sw *generator.Snip
 			if outMember.Type.IsAssignable() {
 				sw.Do("out.$.name$ = $.outType|raw$(in.$.name$)\n", args)
 			} else {
-				if g.convertibleOnlyWithinPackage(m.Type, outMember.Type) {
+				if g.preexists(m.Type, outMember.Type) {
```

Comment:
You could return the function as well, then this would be something like `if function, ok := preexists(m.Type, outMember.Type); ok {`. It just seems a tiny bit odd to call this function and then immediately on the next line do exactly the thing this function does.